guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -495,6 +502,9 @@ A CHANGELOG entry should summarize what was changed and should end with the auth
 Your name can be added directly after the last word if there are no code
 examples or multiple paragraphs. Otherwise, it's best to make a new paragraph.
 
+### Ignoring files created by your editor/IDE
+Some editors and IDEs will create a hidden folder inside the `rails` folder when you use them to write your code. Instead of manually removing them from every commit or adding them to Rails' `.gitignore`, you should configure a [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) in your computer.
```

Comment:
```suggestion
Some editors and IDEs will create hidden files or folders inside the `rails` folder. Instead of manually excluding those from each commit or adding them to Rails' `.gitignore`, you should add them to your own [global gitignore file](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).
```

It would be nice if the [official `gitignore` documentation](https://git-scm.com/docs/gitignore) had a direct example to link to like the GitHub page does.  Also, I wish the GitHub used (or at least mentioned) `$XDG_CONFIG_HOME/git/ignore`.